### PR TITLE
Feed

### PR DESCRIPTION
**Pull request by bossmonkey98**
post component added
like dislike post functionality added
Delete post of the user by user functionality added
Custom loader defined
api calls for posts are defined

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

File: src/components/Avatar/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export const Avatar = () => {
+    return (
+        <section className='avatar-container'>
+            <div className='avatar'>
+                <img id='img' src="https://images.unsplash.com/photo-1503023345310-bd7c1de61c7d" alt="" />
+            </div>
+            <span>Bossmonkey</span>
+        </section>
+    )
```
Comment (mayuresh-tech) [inline src/components/Avatar/Avatar.jsx]: Can you make this dynamic?
Or make something for default set

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: src/components/Avatar/Avatar.jsx (hunk 1 above)
Comment (bossmonkey98) [reply src/components/Avatar/Avatar.jsx]: Its just for UI reference haven't worked on Avatar yet

**Pull request merged**